src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 400;
+}
+
+html {
+  font-weight: 300;
+}
+
+.search-bar__input {
+  width: 100%;
+  display: flex;
+  border: 1px solid rgba(225, 232, 237, 1);
+  box-sizing: border-box;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  color: rgba(61, 78, 97, 1);
+  box-shadow: 0 1px 8px #3d4e611a;
+  font-style: normal;
+  border-radius: 4px;
+}
+
+.search-bar__input--big {
+  height: 70px;
+  background-position: 26px 25px;
+  background-size: 19px 19px;
+  font-size: 16px;
+  padding-left: 62px;
+  margin-top: 20px;
+  color: rgba(61, 78, 97, 1);
+}
+
+.search-bar__input--big:hover {
+  box-shadow: 0 4px 4px #00000040;
+}
+
+.searc-bar__input--big:focus {
```

Comment:
There seems to be a typo in the selector name '.searc-bar__input--big:focus'. It should probably be '.search-bar__input--big:focus'.